tracks/go/exercises/nucleotide-count/mentoring.md

```diff
@@ -0,0 +1,98 @@
+# Nucleotide Count
+
+### Reasonable Solutions
+
+Using a map is probably the most idiomatic solution to this problem, especially since the Histogram type basically must be a map. Using a switch is about 4 times faster over the provided test cases because it does not require hashing.
+
+#### Map Example
+```Go
```

Comment:
I thought language tags needed to be all lowercase to work. Is that incorrect?

Reply:
It looks like it's titlecase in the [language file GitHub points to](https://github.com/github-linguist/linguist/blob/master/lib/linguist/languages.yml#L2470). Or maybe it's not case sensitive?